packages/account/src/Components/self-exclusion/self-exclusion-article.tsx

```diff
@@ -9,12 +9,11 @@ import SelfExclusionContext from './self-exclusion-context';
 const SelfExclusionArticle = () => {
     const { is_app_settings, toggleArticle, is_eu, is_uk } = React.useContext(SelfExclusionContext);
     const { is_deriv_crypto } = React.useContext(PlatformContext);
-    const { is_appstore } = React.useContext(PlatformContext);
     return (
         <AccountArticle
             title={localize('Trading limits and self-exclusion')}
             descriptions={
-                is_appstore && isDesktop()
+                isDesktop()
```

Comment:
can you pull the latest changes, we have included `is_desktop` in ui we can use that instead of isDesktop 😎